### ISSUE TITLE
RSS feed support

### ISSUE THREAD

**Issue by fazilaltinel** [Pseudomanifold/DONUT#4]
Thank you for the great project. It is very useful for discovering papers on TDA.

I would like to know if you plan to add RSS feed support to the platform. It would be quite useful if I can follow new papers from my RSS feed aggregator.

Fazil

**Comment by Pseudomanifold** [comment]
Thanks! I think RSS could be accomplished quite easily. From our side, we need to ensure that the identifiers of the respective articles are persistent (no pun intended). Maybe @yossibokorbleile and I could meet about this soon?

**Comment by yossibokorbleile** [comment]
@Pseudomanifold yes! Sounds like a great idea. I think we are overdue for a meeting anyway.

**Comment by fazilaltinel** [comment]
Thank you guys for your swift replies. I am looking forward to using the feature.

**Comment by Pseudomanifold** [comment]
Some updates on this: the database is now queried and stored in the repository, since it's all based on text anyway. We can now start thinking about feed generation.
